src/jit/lsra.cpp

```diff
@@ -4115,6 +4115,8 @@ LinearScan::setFrameType()
         }
     }
 
+    frameType = FT_EBP_FRAME;
+
```

Comment:
Temporary workaround to get JIT\Regression\JitBlue\DevDiv_200492\DevDiv_200492\DevDiv_200492.cmd working, see explanation a few posts above. The correct solution is to detect when an EBP frame is required for x86. Probably I should create an issue for this.

Obviously, those workarounds need to be removed if this is to be merged.